frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/resource-status.tsx

```diff
@@ -23,7 +23,7 @@ type ResourceStatusProps = {
  * )
  * ```
  */
-const ResourceStatus: React.FC<ResourceStatusProps> = ({
+export const ResourceStatus: React.FC<ResourceStatusProps> = ({
```

Comment:
@fhlavac why are we not using the default export we have below?

Reply:
Good point, I will fix that